src/engine/SCons/Tool/MSCommon/vc.py

```diff
@@ -355,15 +355,30 @@ def cached_get_installed_vcs():
 
 def get_installed_vcs():
     installed_versions = []
+
+    def clfind(name, path):
+        '''Search for a filename in a given path.
```

Comment:
Is the intent to search recursively under a specified directory for cl.exe? or just that directory? If recursive, please indication the docstring.

Reply:
Recursive... the 2017 stuff puts things much deeper down than the earlier versions.  I'll wait a little more before updating in case there are other comments.